jrnl/util.py

```diff
@@ -4,21 +4,30 @@
 import os
 import getpass as gp
 import yaml
+import colorama
 if "win32" in sys.platform:
-    import colorama
     colorama.init()
 import re
 import tempfile
 import subprocess
 import unicodedata
 import shlex
+from string import punctuation, whitespace
 import logging
 
 log = logging.getLogger(__name__)
 
-WARNING_COLOR = "\033[33m"
-ERROR_COLOR = "\033[31m"
-RESET_COLOR = "\033[0m"
+PY3 = sys.version_info[0] == 3
+PY2 = sys.version_info[0] == 2
```

Comment:
Don't add all these constants back in, they were purposefully removed previously.

Reply:
I also meant the ones below this comment (only excluding the colors)

Reply:
Whoops. Fixed.